AlphaWallet/Tokens/Coordinators/ClaimOrderCoordinator.swift

```diff
@@ -66,8 +67,15 @@ class ClaimOrderCoordinator {
                              v: UInt8,
                              r: String,
                              s: String,
+                             recipient: String,
                              completion: @escaping (Result<String, AnyError>) -> Void) {
-        let request = ClaimERC875Spawnable(tokenIds: tokenIds, v: v, r: r, s: s, expiry: expiry)
+        var tokenStrings = [String]()
```

Comment:
This is a bit shorter:

```
var tokenStrings: [String] = tokenIds.map { MarketQueueHandler.bytesToHexa($0.serialize().array) }
```

But this version is terser and clearer if that's what you are trying to do:

```
var tokenStrings: [String] = tokenIds.map { String(tokenIds[0], radix: 16) }
```

Maybe it's better to just pass tokenIds directly to ClaimERC875Spawnable (like before this PR) and move the conversion to [String] there instead? Based on the comment "conversion from BigUInt fails in abi encoding", marshalling the arguments is its responsibility